### DISCUSSION TITLE
EGFX, nVidia questions

### DISCUSSION THREAD

**Discussion by philstopford** [neutrinolabs/xorgxrdp#297] [Q&A]
I run Arch and have the xorgxrdp/xrdp set up running nicely, but with the latest package 0.10 I'm seeing llvmpipe as the renderer reported by glxinfo when the server has nVidia hardware. I had previously played with the EGFX branch/fork and there I would see the nVidia hardware detected and used.

I just wanted to ask about what might be needed here. Do I need to build xrdp/xorgxrdp locally with some specific parameters to get the nVidia hardware available for acceleration, or would this indicate a bug somewhere (packaging or setup)?

**Comment by UnderADome** [comment]
to see xrdp project pull/3320 and issues/1697, you will figure out what to do
